Client/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/PlaylistScript.js

```diff
@@ -230,6 +234,14 @@ window.__firefox__.includeOnce("Playlist", function($) {
         })
       }, 2000);  // Resolution of 1000ms is fine for us.
     }
+
+    function onReady(fn) {
+      if (document.readyState === "complete" || document.readyState === "ready") {
+        setTimeout(fn, 1);
```

Comment:
Resolved. I wanted to just run it with a slight delay so it runs on the next browser run-loop. But actually it is fine running immediately since the task is `posted`